content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -0,0 +1,130 @@
+
+*Bundled content* are product assets that're bundled from the project's `Cache/pc` directory into package files (`.pak`). The Game Launcher loads the bundled content that make up a project, such as its levels, objects, environment, and gameplay logic.
+
+The instructions here guide you through the following steps:
+- Set the starting level.
+- Process your project's assets.
+- Create a project game release layout.
+- Run your project's Game Launcher.
+
+
+## Prerequisites
+
+The following instructions assume that you have:
+
+- Set up O3DE on your computer. For help, refer to [Set up Open 3D Engine](/docs/welcome-guide/setup/).
+
+- Created an O3DE project that contains at least one level. You may need to resolve any errors in your project in order to create a release build.
+
+- Generated a Visual Studio project for your O3DE project. For help, refer to [Create a Visual Studio project](/docs/welcome-guide/create/creating-projects-using-cli/#create-a-visual-studio-project).
+
+
+This tutorial uses the following project name and directories in the examples.
+
+- Project name and location: `C:\MyProject`
+- Third-party directory location: `C:\o3de-packages`
+
+
+## Set the starting level
+
+Before you create a project game release layout, specify the starting level that the Game Launcher will load first in your settings registry.
+
+1. Create an `autoexec.game.setreg` file in `MyProject/Registry` directory and add the following lines. This sets the starting level by using `Loadlevel` and providing a path to the level. The path must be relative to the `Cache/pc` directory. In this example, the level asset is `mainmenu.spawnable`.
+
+
+```
+{
+    "O3DE": {
+        "Autoexec": {
+            "ConsoleCommands": {
+                "LoadLevel": "levels/mainmenu/mainmenu.spawnable"
+            }
+        }
+    }
+}
+```
+
+
+## Process assets
+
+The Asset Processor or Asset Processor Batch are responsible for processing source assets into product assets, which the Game Launcher consumes at runtime.
+
+1. Process your project's assets by following either one of the following instructions.
+    
+    - Use **CMake** to build the Asset Processor Batch and its dependencies by specifying the `MyProject.Assets` target. After building, this command runs Asset Processor Batch and processes the assets.
+
+        ```cmd
+        cmake --build build/windows_vs2019 --target MyProject.Assets --config profile -- /m
+        ```
+
+    - Run your project's Asset Processor to begin processing the assets.
+
+
+{{< caution >}} Ensure that the Asset Processor or Asset Processor Batch processes all of your assets without failure. {{< /caution >}}
+
+
+## Create a project game release layout
```

Comment:
it shoudl be specified that the monolithic project can be built only againt the monolithic SDK. There are no steps mentioend or linked that talk about building monolithic SDK

Reply:
Well the monolithic SDK isn't required to build a monolithic project.

It is more accurate to state that a a project that uses an SDK, can only build monolithically if the SDK was built with monolithic support.

A project that uses the engine-source can be build monolithically without an SDK.
By using a project-centric engine-source workflow
`project-centric = CMake source directory is the project root`
`engine-source = The engine registered with the project refers to a location where the engine source code resides`

Reply:
I'll explain the differences in workflows and non-monolithic/monolithic in detail here

First we are going to assume the engine source repo is located at `C:/o3de`.
The created SDK layout be located at `C:/o3de-install`
Any the project that we will create is located at `C:/TestProject`

So first the user grabs the O3DE repo from github.

At this point they have the following options available them.
- They can create a new projects using the `o3de.py` script
- They can register the engine with o3de_manifest.json
- They can register the engine with any existing projects and optionally build those projects using a project-centric engine-source workflow (`cmake -S C:/TestProject ...`)
- They can build the engine using an engine-centric workflow **non-monolithic(`cmake** -S C:/o3de -DLY_MONOLITHIC_GAME=0 ...`)
- They can build the engine using an engine-centric workflow **monolithic(`cmake** -S C:/o3de -DLY_MONOLITHIC_GAME=1 ...`)
- They can "append" to SDK layout using either of the two previous previous bullet points

What the user _can't_ do at this point is to perform a project-centric engine-SDK build because they don't have an SDK yet.

So the steps for a user to create a project game release layout using a project-centric engine-source workflow could be as follows.

Non-Monolithic
---
```
# Create a project (This automatically registers the project with the engine where the o3de.py scripts resides)
C:/o3de>scripts\o3de.bat --create-project -pp C:/TestProject
# Configure the project-centric (by specifying the project as the CMake source directory) engine-source(via using the C:/o3de engine) workflow

# Either one of the following two commands are equivalent when it comes to cmake
C:/ThisPathDoesNotMatterItCanBeAnything>cmake -B C:/TestProject/build/windows_non_mono -S C:/TestProject
# or
C:/TestProject>cmake -B build/windows_non_mono -S .

# Make sure to process the Assets.
# This can be done at any point after the project has been created by running the AssetProcessor(Batch)
# But for this exercise the command to process the Assets will be added
# Every project has a ${ProjectName}.Assets CMake target, that builds the AssetProcessorBatch
# and executes it
Z:/ThisPathDoesNotMatterItCanBeAnything>cmake --build C:/TestProject/build/windows_mono --target TestProject.Assets --config release

# Next the Project Game Release Layout can be built using the following command to have CMake forward the build arguments to the underlying build tool
Z:/ThisPathDoesNotMatterItCanBeAnything>cmake --build C:/TestProject/build/windows_non_mono --target INSTALL --config release

# After the build command has run the Project Game release layout should be located at C:/TestProject/nstall/bin/Windows/release/Default
# The game launcher can be run with the following command now
C:/TestProject/install/bin/Windows/release/Default>TestProject.GameLauncher.exe
```

That covers the non-monolithic steps.

For the monolithic steps with using a project-centric engine-source workflow the only difference is the `LY_MONOLITHIC_GAME` variable value specified when configuring cmake

Monolithic
---
```
# Create a project (This automatically registers the project with the engine where the o3de.py scripts resides)
C:/o3de>scripts\o3de.bat --create-project -pp C:/TestProject
# Configure the project-centric (by specifying the project as the CMake source directory) engine-source(via using the C:/o3de engine) workflow

# Either one of the following two commands are equivalent when it comes to cmake
C:/ThisPathDoesNotMatterItCanBeAnything>cmake -B C:/TestProject/build/windows_mono -S C:/TestProject -DLY_MONOLITHIC_GAME=1
# or
C:/TestProject>cmake -B build/windows_mono -S . -DLY_MONOLITHIC_GAME=1

# Make sure to process the Assets.
# This can be done at any point after the project has been created by running the AssetProcessor(Batch)
# But for this exercise the command to process the Assets will be added
# Every project has a ${ProjectName}.Assets CMake target, that builds the AssetProcessorBatch
# and executes it
Z:/ThisPathDoesNotMatterItCanBeAnything>cmake --build C:/TestProject/build/windows_mono --target TestProject.Assets --config release

# Next the Project Game Release Layout can be built using the following command to have CMake forward the build arguments to the underlying build tool
Z:/ThisPathDoesNotMatterItCanBeAnything>cmake --build C:/TestProject/build/windows_mono --target INSTALL --config release

# After the build command has run the Project Game release layout should be located at C:/TestProject/nstall/bin/Windows/release/Monolithic
# The game launcher can be run with the following command now
C:/TestProject/install/bin/Windows/release/Monolithic>TestProject.GameLauncher.exe
```

That covers how to create a project game release layout without using an SDK.


### Now to create a Project Game Release layout using an SDK, the monolithic build artifacts will have to be added to the SDK.

We can use the O3DE repo at `C:/o3de` to create an SDK layout with both non-monolithic and monolithic artifacts.

Earlier in the post, one of the options available to the user was
> They can "append" to SDK layout using either of the two previous previous bullet points

The keyword here is "append".

Now on to the steps to create an SDK layout with both Non-Monolithic and Monolithic artifacts
NOTE: This is engine-centric; The current directory is the engine root and the CMAKE source directory specified via the the `-S` option supplies the current directory `.` as the path

SDK layout "append" Non-Monolithic artifacts
---
```
# First customize the SDK layout directory using the -DCMAKE_INSTALL_PREFIX option and the new engine name using the -DLY_VERSION_ENGINE_NAME option
C:/o3de>cmake -B build/windows_vs2019 -S . -DCMAKE_INSTALL_PREFIX=C:/o3de-install -DLY_VERSION_ENGINE_NAME=o3de-install

# Next add the profile non-monolithic artifacts will be appended to the SDK layout
C:/o3de>cmake --build build/windows_vs2019 --target INSTALL --config profile

# Now add the release non-monolithic artifacts will be appended
C:/o3de>cmake --build build/windows_vs2019 --target INSTALL --config release
```

This will create an SDK layout at `C:/o3de-install` that has both the `profile` and `release` non-monolithic directories
This can validated checking that the following directories exists `C:/o3de-install/lib/Windows/profile/Default` and `C:/o3de-install/lib/Windows/release/Default`

Next is the steps that follow will append the Monolithic artifacts to the existing SDK layout

SDK layout "append" Monolithic artifacts
---
```
# Point the monolithic build folder at the same the SDK layout folder as the non-monolithic configuration
C:/o3de>cmake -B build/windows_mono -S . -DLY_MONOLITHIC_GAME=1 -DCMAKE_INSTALL_PREFIX=C:/o3de-install -DLY_VERSION_ENGINE_NAME=o3de-install

# Now we will only add the release monolithic artifacts to the SDK layout. Adding profile artifacts doesn't really help create a Project Game Release Layout
C:/o3de>cmake --build build/windows_mono --target INSTALL --config release
```

At this point the SDK layout at `C:/o3de-install should contain the `release` monolithic directories.
This can be verified by checking that the `C:/o3de-install/lib/Windows/release/Monolithic` directory exists


Finally at this point the SDK layout can be used to create a Project Game Release Layout using either non-monolithic or monolithic artifacts.
Since this post is so long already, I'll just show the commands to create a Project Game Release Layout with a monolithic game launcher

Make sure python is installed and the engine is registered with `o3de_manifest.json` beforehand

```
C:/o3de>python\get_python.bat
C:/o3de>scripts\o3de.bat register --this-engine
```

Also make sure the Assets are already processed for the project

Monolithic
---
```
# Use the SDK layout `o3de.py` script to register the SDK engine with the Test Project
# NOTE: The current directory is the engine root
C:/o3de>scripts\o3de.bat register --pp C:/TestProject

# Generate a project-centric monolithic build solution. 
# NOTE: the current directory is the project root
C:\TestProject>cmake -B build/windows_mono_install -S . -DLY_MONOLITHIC_GAME=1

# Generate the Project Game Release layout by building the INSTALL target in the release config
C:\TestProject>cmake -B build/windows_mono_install --target INSTALL --config release

# The game launcher can also now be run with the following command
C:/TestProject/install/bin/Windows/release/Monolithic>TestProject.GameLauncher.exe
```

### Side Note:
The `INSTALL` command is actually called "install" when using `ninja` or `make` as a generator.
Those generators are normally used on Linux